### ISSUE TITLE
@searchable in v1.12.0 sorting on createdAt & updatedAt fields gives an ElasticSearch bad request error

### ISSUE THREAD

**Issue by nino-moreton** [aws-amplify/amplify-cli#2161]
**Describe the bug**
When upgrading to v1.12.0 from v1.11.0, my `@searchable`s that sort on `createdAt` or `updatedAt` gives a bad request response like so
```json
{
  "data": {
    "search********": null
  },
  "errors": [
    {
      "path": [
        "search******"
      ],
      "data": null,
      "errorType": "Elasticsearch:UserIllegalArgumentException",
      "errorInfo": null,
      "locations": [
        {
          "line": 10,
          "column": 3,
          "sourceName": null
        }
      ],
      "message": "Elasticsearch responded with an error: Bad Request"
    }
  ]
}
```

**To Reproduce**
1. In v1.12.0 version of Amplify, create a grapql api with a model that has a @searchable directive. Make sure to expose the `createdAt` & `updatedAt` fields
2. query that searchable model and sort in either `createdAt` or `updatedAt` field. Doesn't matter what directiton
3. See error in response

**Expected behavior**
Should sort correctly like in v1.11.0

**Additional context**
This used to work in the previous version(v1.11.0) of Amplify and out of nowhere with no warning at all, it just stops in the latest version(v1.12.0)

**Comment by nino-moreton** [comment]
I've actually been following that thread @houmark and seems I missed out on the ElasticSearch sorting issue part. As per @kaustavghosh06 's response this issue should be fixed once PR #2090 is released. Thank you! 

Hoping Amplify team will address their project management problem in that thread as I think that was the main point of your opening that issue.

**Comment by yuth** [comment]
We have fixed this in #2090
